src/store/modules/tester/codeToPos.js

```diff
@@ -0,0 +1,213 @@
+export const ANSI = [
```

Comment:
I think they differ by 2 positions. Let's just take ANSI and slice it to make ISO?